test/blackbox-tests/test-cases/dune-project-meta/github2927/run.t

```diff
@@ -0,0 +1,16 @@
+Generation of opam files is attached to @all
+--------------------------------------------
+Reproduction case for #2927
+
+  $ mkdir attached-to-all
+  $ cat >attached-to-all/dune-project <<EOF
+  > (lang dune 2.0)
+  > (generate_opam_files true)
+  > (package (name foo))
+  > EOF
+
+  $ cd attached-to-all && dune build
+
+  $ cat attached-to-all/foo.opam
+  cat: attached-to-all/foo.opam: No such file or directory
+  [1]
```

Comment:
Does this really fix the issue? I feel like the opam file should be generated here, not missing

Reply:
Good catch. It was a bug in the test itself.